packages/utils/src/UserID.ts

```diff
@@ -2,6 +2,12 @@ import { EthereumAddress, toEthereumAddress } from './EthereumAddress'
 
 export type UserID = EthereumAddress
 
-export const toUserId = (value: string): UserID => {
-    return toEthereumAddress(value)
+const REGEX = /^0x[a-fA-F0-9]+$/
+
+export const toUserId = (input: string): UserID => {
+    return toEthereumAddress(input)
```

Comment:
toEthereumAddress intentional? Then userId has to be specific length?

Reply:
This is pre-https://github.com/streamr-dev/network/pull/2774 era functionality.